Makefile

```diff
@@ -68,4 +68,7 @@ docker-build-and-push: docker-build docker-push
 
 test: e2e-test compatibility-test license-test go-report-card-test helm-sync-test
 
+unit-test: 
+	go test ./... -v -coverprofile=build/coverage.txt -covermode=atomic
```

Comment:
use the `${BUILD_DIR_PATH}/coverage.txt`

Reply:
also use `go test ${MAKEFILE_PATH}/...` so that we can reference the Makefile from outside the root like this: `make -f ../../Makefile unit-test`